src/components/common/NameRequestInfo.vue

```diff
@@ -364,17 +372,18 @@ p {
 }
 
 ul {
-  font-size: $px-14;
+  font-size: $px-15;
```

Comment:
This is closer to the UI comp without looking weird (due to other font sizes not being exactly according to the spec). This was an easy fix. Doing all isn't easy.